### PR TITLE
fix(examples): updating example microcopy

### LINKED ISSUE #5714: Microcopy revisions needed across component docs
Content team has identified microcopy changes needed in doc examples:

- [x] About modal: 
  - React: 
    - Change buttons to sentence case (Show About Modal > Show about modal)

- [x] Accordion: 
  - React:
    - Change items to sentence case (Item One > Item one)

- [x] Alert: 
  - React:
  	- In the “Static live region alert” section, make “alert” lowercase (Alert > alert).
	- Change buttons to sentence case (e.g., Add Single Info Alert >  Add single info alert)
	- Rewrite the first static live region alert so that the component name is lowercase and fix subject/verb agreement between “prop” and “sets”:  
“This alert uses the recommended `isLiveRegion` prop to automatically set ARIA attributes and CSS classes.”
	- Rewrite async info alerts to “This is an async alert number 1,” “This is async alert number 1,” or “Async alert number 1” to introduce each async alert with the correct article (a/an).

- [x] Alert group:
  - React:
	- Change static alert group types to sentence case: “Success alert” and “Info alert.”
	- Change all buttons to sentence case for each remaining section (e.g., “Add Toast Success Alert” > “Add toast success alert”)

- [x] Application launcher:
  - React:
  	- Change all instances of “Unavailable Application” to sentence case (“Unavailable application”).

- [x] Breadcrumb:
  - React:
    - Change “Section Landing” to sentence case (“Section landing”).

- [x] Card:
  - React:
  	- Edit the only actions in card head description for subject/verb agreement: “This is the card body. There are only actions in the card head.” 
	- Change selectable card copy to: “This is a selectable card. Click to select. Click again to deselect.” 

- [x] Card view demo:
  - Demos (under Demos section in nav):
    - Rewrite “cards” in lowercase: “This is a demo that showcases PatternFly cards.”
	- Rewrite all instances of “Patternfly” as “PatternFly.”
Change button to sentence case and remove article: “Create a Project” > “Create project”

- [x] Chip group: 
  - React:
    - In “Single” section, make all instances of “chip” lowercase if it’s not the first word in the chip (e.g., “Really long chip that…”).

- [x] Clipboard copy:
  - React:
    - Change “copy to clipboard” tooltip to: “Copy” and “Copied” (instead of “Copy to clipboard” and “Successfully copied to clipboard!”)

- [x] Code editor: 
  - React:
    - Change “Display Minimap” to “Minimap” so that items follow parallel structure. 
	- Change button from “Browse” to “Upload.”
	- ~~Change second line of empty state to: “Drag and drop a file or upload one.”~~
	  - split out into issue https://github.com/patternfly/patternfly-react/issues/6195

- [x] Context selector: 
  - React:
    - Change dropdown menu selections to sentence case.

- [x] Date picker: 
  - React: 
    - ~~Change basic format to: YYYY-MM-DD~~
    - ~~Change American format date to: 03-05-2020~~
      - Above 2 items moved into [#6199](https://github.com/patternfly/patternfly-react/issues/6199) for follow up discussion
    - Change helper text to “Select a date.”  
  - React demos: 
    - Make same changes as React.
    - Change description to: “This is intended to be used as a filter. After selecting a start date, the next date is automatically selected.”

- [x] Drawer:
  - React:
    - Change buttons to sentence case.

- [x] Dropdown:
  - React:
    - Change all dropdown menu items to sentence case.

- [x] Empty state: 
  - React:
    - Change headline and buttons to sentence case.  
	- Edit “No match found” empty state second line to: “Clear all filters and try again.”

- [x] Expandable section: 
  - React:
    - “Show more” and “Show less” should be in sentence case.

- [x] File upload:
  - React:
    - Change text field microcopy to “Drag and drop a file or upload one” 
	- Change “Browse…” to “Upload” 

- [x] Form: 
  - React:
    -  Change the “Name” field to “Full name” 
	- Remove helper text: “Please provide your full name” 
	- Add a period after “Please don’t contact me.” 
	- Change “Additional Note” to sentence case: “Additional note” 
	- Add period after “I’d like updates via email.” 
	- Change “Submit form” button to “Submit” 
	- Change “Please Choose” dropdown to “Select one” 
	- Add period after “Follow up via email.” and “Remember my password after 30 days.” 
	- Change “Age” error text to: “Must be a number” (and omit period) 
	- Change red danger alert to: “Fill out all required fields before continuing.”  
	- Add a colon after “Select all that apply:” Make the “o” in each option uppercase: Option 1; Option 2

- [x] Form select:
  - React:
    - Change “Choose” to “Select” to match UX writing style guide’s recommendations. 
	- Remove “Make a selection” helper text.  
	- Change “Please Choose” in Basic dropdown to “Select one” 
	- Change dropdown options in “Grouped” to sentence case.

- [x] Input group:
  - React:
    - Punctuate popover content and add articles: “This field is an example of an input group with a popover.”

- [x] Label group:
  - React:
    - For consistency, use numbers in “Category removable” and “Vertical category overflow removable” sections (e.g., “Label 1”).

- [x] Login page:
  - React:
    - Change “Please use your single sign-on LDAP credentials” to “Enter your single sign-on LDAP credentials.”
	- Change button to sentence case.

- [x] Menu:
  - React: 
    - Change all menu items to sentence case.

- [x] Modal:
  - React:
    - Change all buttons to sentence case.

- [x] Navigation:
  - React demos:
    - Fix typo “it’s” to “its” in page description. (Is this sentence supposed to wrap?)

- [x] Notification drawer:
  - React:
    - Change kebab menu items in “Groups” section to sentence case.
  - React demos:
    - Fix typo “it’s” to “its” in page description. (Is this sentence supposed to wrap?)

- [x] Page:
  - React: 
    - Change nav item titles to sentence case.
  - React demos:
    - Fix typo “it’s” to “its” in page description. (Is this sentence supposed to wrap?)
    - Change nav items titles to sentence case.

- [x] Popover:
  - React: 
    - Change all buttons to sentence case.
    - Change “Popover” to “popover” in the “Close popover from content (uncontrolled)” section.
	- Change “Popover Header” and “Popover Footer” to sentence case in ref sections.
	- Change “Popover Position” to sentence case in “Advanced” section.

- [x] Select:
  - React:
    - Change “Choose…” to “Select a title”
    - Change dropdown menu items to sentence case.

- [x] Skip to content:
  - React:
    - Change button to sentence case.

- [x] Slider:
  - React:
    - Change slider value statements to sentence case.

- [x] Table:
  - React:
    - Rewrite empty state’s second line to “Clear all filters and try again.” 
  - React demos:
    - Fix typo “it’s” to “its” in page description. (Is this sentence supposed to wrap?)
	- Rewrite “Unable to connect” empty state second line to “Check your connection and reload the page.” 
	- Rewrite “No results found” empty state second line to “Clear all filters and try again.”  
	- Rewrite last empty state’s second line to “Check your connection and reload the page.”

- [x] Text area:
  - React:
    - Rewrite helper text to “Share your thoughts.” (include period)

- [x] Toolbar:
  - React demos:
    - Change tooltip text and dropdown menu items to sentence case.

- [x] Wizard:
  - React:
  	- Edit validation screen in “Finished” section:
	  - Change both validation headings to sentence case and align example with forthcoming design guidelines: 
“Validating credentials” and “Validation complete”
	  - Replace Lorem Ipsum with how description should be leveraged: “Description can be used to further elaborate on the validation step, or give the user a better idea of how long the process will take.”
	- Rewrite helper text in “Validate on button press” section: “Write your age in numbers.”
	- Change step names and radio button options to sentence case in “Progressive steps” section.

### PR DESCRIPTION
**Pull request by evwilkin**
**What**: Closes #5714 

This PR updates microcopy across examples & demos as laid out in issue #5714

### CODE REVIEW

File: packages/react-core/src/components/DatePicker/examples/DatePicker.md
Comment (redallen) [inline packages/react-core/src/components/DatePicker/examples/DatePicker.md:30]: If you modify `dateFormat` you'll also have to modify `dateParse`.

BTW, why is our microcopy suggesting the American date format use `-` instead of `/` as the delimiter?

Comment (afialkoff) [reply packages/react-core/src/components/DatePicker/examples/DatePicker.md:30]: Great catch! We recommended **MM-DD-YYYY** based on the existing microcopy guidelines provided for date and time formats on our PatternFly numerics page (https://www.patternfly.org/v4/ux-writing/numerics#date-and-time-formats). 

What do you think is the best solution? We could change the `-` to `/` format here and update the Numerics page to match, but we'd need to make sure we carry that through to Red Hat products that have existing date formats written as **MM-DD-YYYY**.

Curious to hear your thoughts on this too, @abigaeljamie!

Comment (redallen) [reply packages/react-core/src/components/DatePicker/examples/DatePicker.md:30]: Hm, I think the PatternFly numerics page should reflect the international choice of `YYYY-MM-DD` that was signed off by design for the default DatePicker implementation. That's what [ISO8601](https://www.iso.org/iso-8601-date-and-time-format.html) and the [ECMA standard](https://262.ecma-international.org/11.0/#sec-date.parse) use.

Comment (abigaeljamie) [reply packages/react-core/src/components/DatePicker/examples/DatePicker.md:30]: If design signed off on the international formatting (YYY-MM-DD) for the DatePicker implementation, I'm fine with changing the Numerics page to reflect that guidance. Would it be possible to change the component to reflect that, @evwilkin ?

Comment (evwilkin) [reply packages/react-core/src/components/DatePicker/examples/DatePicker.md]: @abigaeljamie @afialkoff @redallen I'm going to pull these date formatting changes out of this PR for now, it sounds like a bigger discussion on not just how to proceed with this microcopy but with the guidelines for users and how changing this guidance will affect our users.  Created https://github.com/patternfly/patternfly-react/issues/6199 as a follow-up.

**Review by redallen — APPROVED** [review]
Sometimes no change is a good change :)

**Review by tlabaj — APPROVED** [review]
LGTM. We can merge once it get Abi's approval

**Review by abigaeljamie — APPROVED** [review]

**Pull request merged**
